util/common_test.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util
+
+import (
+	"testing"
+)
+
+func TestIsValueNil(t *testing.T) {
+	if !IsValueNil(nil) {
```

Comment:
Any reason not to make this test function table-driven per other tests?

Reply:
The reason is that it's not possible in this case to have an input field to share across the tests. It would have to be cast to interface{} and we would not get the same behavior.

Reply:
Ah, I'd missed that the arguments here weren't `interface{}`. Thanks.